neural_compressor/torch/algorithms/smooth_quant/utility.py

```diff
@@ -1198,50 +1337,63 @@ def _get_one_batch_auto_loss(self, input, alpha_space, orig_best_alpha, input_ma
                 output = module.q_dq_forward(module.q_input, module.input_scale, module.weight_scale)
                 loss = self._get_auto_loss(fp32_output[name], output)
                 loss_alphas[name][str(alpha)] = loss
+
         return loss_alphas
 
-    def _get_one_batch_auto_loss_blockwise(self, input, alpha_space, orig_best_alpha, input_maxes):
+    def _get_one_batch_auto_loss_blockwise(
+        self, input, alpha_space, orig_best_alpha, input_maxes, calib_func, calib_sample_num
+    ):
         """Calculate the losses for all alpha values given an input in blockwise tuning mode.
 
         :return: A dict of blockwise-wise loss values with respect to alpha values.
         """
         self._change_qdq_for_auto(enable=False)
-        module_names = self._get_sq_layer_names()
 
         block_modules = {}
         for key in self.block_names:
             block_modules[key] = get_module(self.model, key)
         self._add_blockwise_observer(block_modules)
 
-        forward_wrapper(self.model, input, self.device)  ##disable quant and get fp32 output
+        # get input args and kwargs for the first block, then do forward
+        total_block_args, total_block_kwargs = get_hidden_states(self.model, calib_sample_num, calib_func)

```

Comment:
seems like we fused all output into once instead of per input? Will it impact the final result @yintong-lu